core-android/src/main/java/com/uber/sdk/android/core/auth/LoginActivity.java

```diff
@@ -453,4 +573,25 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
             return super.shouldOverrideUrlLoading(view, url);
         }
     }
+
+    class LoginPARCallback implements LoginPushedAuthorizationRequest.Callback {
+
+        private final ResponseType responseType;
+
+        LoginPARCallback(ResponseType responseType) {
+            this.responseType = responseType;
+        }
+
+        @Override
+        public void onSuccess(String requestUri) {
+            removeProgressIndicator();
+            loginInternal(requestUri);
+        }
+
+        @Override
+        public void onError(LoginPARRequestException e) {
+            removeProgressIndicator();
+            loginInternal("");
```

Comment:
Should we be continuing in login flow if there's an exception here?

Reply:
yes, we don't want to block them here. Instead the login flow will continue without the profile prefill